rdt-app/src/main/java/io/ona/rdt_app/activity/CustomRDTCaptureActivity.java

```diff
@@ -0,0 +1,36 @@
+package io.ona.rdt_app.activity;
+
+import android.os.Bundle;
+import android.util.Log;
+
+import edu.washington.cs.ubicomplab.rdt_reader.ImageProcessor;
+import edu.washington.cs.ubicomplab.rdt_reader.activity.RDTCaptureActivity;
+import io.ona.rdt_app.BuildConfig;
+import io.ona.rdt_app.application.RDTApplication;
+import io.ona.rdt_app.presenter.CustomRDTCapturePresenter;
+
+import static io.ona.rdt_app.util.RDTJsonFormUtils.convertByteArrayToBitmap;
+
+/**
+ * Created by Vincent Karuri on 27/06/2019
+ */
+public class CustomRDTCaptureActivity extends RDTCaptureActivity {
+
+    private static final String TAG = CustomRDTCaptureActivity.class.getName();
+
+    private CustomRDTCapturePresenter presenter;
+    private String baseEntityId = BuildConfig.BASE_ENTITY_ID; // todo remove this
```

Comment:
Could you pass the patient baseEntityId here

Reply:
Done.